src/components/org/PositionPicker/components/ItemComponent.tsx

```diff
@@ -1,10 +1,11 @@
 import ItemComponentProps from './itemComponentProps';
-import styles from '../styles.less';
+import { useStyles } from '../style';
```

Comment:
we should try to be consistent with the styling files, `itemComponent.styles.ts`?